frontend/src/shared/components/SearchBar/SearchBar.tsx

```diff
@@ -14,17 +14,19 @@ interface SearchBarProps {
   onSubmit: (value: string) => void;
   defaultValue?: string;
   maxLength: number;
+  size?: "small" | "medium";
```

Comment:
```suggestion
export type VariantSize = "small" | "medium";
  size?:  VariantSize;
```
이렇게 타입을 따로 분리를 했을 때 이점이 만약 large가 추가가 된다고 하면 type에서 한 개만 늘어주면 되는데 지금와 같은 방법은 props, css props도 같이 수정이 되어야 할 것 같아서 관리할 포인트를 줄이기 위해서 type으로 따로 분리하는 방법이 좋아보이는데 아더는 어떻게 생각하시나요?🤔